src/ml_gym/cmd_entrypoint/cmd.py

```diff
@@ -56,81 +64,110 @@ class LoggingConfiguration:
     gs_rest_api_endpoint: str
 
 
-def get_gym_from_environment_config(env_config: Union[MultiProcessingEnvironmentConfig, MainProcessEnvironmentConfig,
-                                                      AccelerateEnvironmentConfig],
-                                    logger_collection_constructable: LoggerConstructableIF,
-                                    gs_restful_api_client_constructable: GridSearchAPIClientConstructableIF,
-                                    num_epochs: int,
-                                    num_batches_per_epoch: int = None,
-                                    accelerator: Accelerator = None) -> Gym:
-
+def get_gym_from_environment_config(
+    env_config: Union[MultiProcessingEnvironmentConfig, MainProcessEnvironmentConfig, AccelerateEnvironmentConfig],
+    logger_collection_constructable: LoggerConstructableIF,
+    gs_restful_api_client_constructable: GridSearchAPIClientConstructableIF,
+    num_epochs: int,
+    num_batches_per_epoch: int = None,
+    accelerator: Accelerator = None,
+) -> Gym:
     if isinstance(env_config, MainProcessEnvironmentConfig):
-        gym = GymFactory.get_sequential_gym(logger_collection_constructable=logger_collection_constructable,
```

Comment:
I see all these code reformatting changes. Could you please disable those in your IDE and undo these changes please?

Reply:
done